packages/react-player/src/dotlottie-context.tsx

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright 2023 Design Barn Inc.
+ */
+
+import type { DotLottiePlayer } from 'common';
+import { createContext, useContext } from 'react';
+
+export const DotLottieContext = createContext<DotLottiePlayer | undefined>(undefined);
+
+export const useDotLottieContext = (): DotLottiePlayer | undefined => {
+  const dotLottiePlayer = useContext(DotLottieContext);
+
+  return dotLottiePlayer;
+};
```

Comment:
I think the `undefined` is unnecessary, cause it means the context value, hasn't been set. and i believe we set it synchronously when rendering the main react-player component, correct ? 

```suggestion
export const useDotLottieContext = (): DotLottiePlayer => {
  const dotLottiePlayer = useContext(DotLottieContext);
  
 if(typeof dotLottiePlayer === 'undefined') throw new Error("useDotLottieContext must be wrapped by the DotLottieContext Provider")

  return dotLottiePlayer;
};
```

Reply:
Umm dotLottiePlayer instance comes from  `useDotLottiePlayer` hook. This hook waits for html `container` element to be ready. I am not sure if this would be the right approach. We can talk about this tomorrow.

Reply:
Done ✅